source/common/common/utility.h

```diff
@@ -551,7 +552,38 @@ template <class Value> struct TrieEntry {
  * A trie used for faster lookup with lookup time at most equal to the size of the key.
  */
 template <class Value> struct TrieLookupTable {
-
+  ~TrieLookupTable() {
```

Comment:
Use a comment to explain why this custom destructor is needed.

Reply:
> Use a comment to explain why this custom destructor is needed.

Added, thanks!